bitcoin/src/address/mod.rs

```diff
@@ -373,9 +373,9 @@ impl Address {
     /// This address type was introduced with BIP16 and is the popular type to implement multi-sig
     /// these days.
     #[inline]
-    pub fn p2sh(script: &Script, network: impl Into<NetworkKind>) -> Result<Address, Error> {
+    pub fn p2sh(script: &Script, network: impl Into<NetworkKind>) -> Result<Address, FromScriptError> {
```

Comment:
This function has a single exit path but the returned error has three variants. Please forgive me for being abrupt but this is literally the example I wrote out for you yesterday.

Reply:
Oh, sorry for this. I remembered to give it a seperate name till I grouped that within the same `FromScriptError`. Recitfying it.